src/pages/configurePatch/ConfigurePatchCore.tsx

```diff
@@ -150,6 +156,43 @@ const ConfigurePatch: React.FC<Props> = ({ patch }) => {
               <Tab data-cy="changes-tab" name="Changes" id="changes-tab">
                 <CodeChanges />
               </Tab>
+              <Tab
+                data-cy="parameters-tab"
+                name="Parameters"
+                id="parameters-tab"
+              >
+                <ParamsContainer>
+                  {patch?.activated ? (
+                    <>
+                      <Disclaimer>
+                        <span data-cy="parameters-disclaimer">
+                          Parameters cannot be added or modified once a patch is
+                          configured
+                        </span>
+                      </Disclaimer>
+                      {patch?.parameters && (
+                        <ExistingParamsContainer>
+                          {" "}
+                          {patch.parameters?.map((param) => (
+                            <StyledBadge
+                              data-cy={`badge-${param.key}`}
+                              key={`param_${param.key}`}
+                            >
+                              {param.key}:{param.value}
+                            </StyledBadge>
+                          ))}
+                        </ExistingParamsContainer>
+                      )}
+                    </>
+                  ) : (
+                    <EditableTagField
```

Comment:
can we fix the max width for this component? it becomes less ergonomic to read at wide screen widths
![Screen Shot 2020-12-16 at 11 12 15 AM](https://user-images.githubusercontent.com/10734386/102375082-12e88180-3f90-11eb-82f5-0b213f04d70a.png)

Reply:
Can we do this in a different ticket? This is a component that has more uses, and Mohamed is more familiar with how it will affect all the use cases.

Reply:
scratch that. I added a max-width of 70% and it seems to work well.